packages/@react-spectrum/s2/style/spectrum-theme.ts

```diff
@@ -74,41 +74,138 @@ const color = {
   LinkText: 'LinkText'
 };
 
-export function baseColor(base: keyof typeof color): {default: keyof typeof color, isHovered: keyof typeof color, isFocusVisible: keyof typeof color, isPressed: keyof typeof color} {
-  let keys = Object.keys(color) as (keyof typeof color)[];
-  let index = keys.indexOf(base);
+// Resolves a color to its most basic form, following all aliases.
+function resolveColorToken(token: string | ColorToken | ColorRef): ColorToken {
+  if (typeof token === 'string') {
+    return {
+      type: 'color',
+      light: token,
+      dark: token
+    };
+  }
+
+  if (token.type === 'color') {
+    return token;
+  }
+
+  let lightToken = baseColors[token.light];
+  if (!lightToken) {
+    throw new Error(`${token.light} is not a valid color reference`);
```

Comment:
```
throw new Error(`${token.light} is not a valid light color reference`);
```
might be hard to know which one is failing if there isn't 1:1 parity

Reply:
If this throws it's a bug in the Spectrum tokens. It won't be caused by user error.